src/octopeer-github/main/Options/DoNotWatchOptions.ts

```diff
@@ -43,18 +40,18 @@ const DoNotWatchOptions = new (class DoNotWatchOptions {
      * @returns {ElementEventBindingCreatable[]}
      */
     public getEvents() {
-        let doNotWatchEvents: ElementEventBindingCreatable[] = [];
+        let doNotWatchEvents: ElementEventBindingData[] = [];
 
         if (Options.getDoNotWatchOnScreenEvents()) {
-            doNotWatchEvents.push(ScrollIntoViewElementEventBinding);
-            doNotWatchEvents.push(ScrollOutOfViewElementEventBinding);
+            doNotWatchEvents.push(eebFactory.findElementEventBindingData(EventID.SCROLL_INTO_VIEW));
+            doNotWatchEvents.push(eebFactory.findElementEventBindingData(EventID.SCROLL_OUT_OF_VIEW));
         }
         if (Options.getDoNotWatchHoverEvents()) {
-            doNotWatchEvents.push(MouseEnterElementEventBinding);
-            doNotWatchEvents.push(MouseLeaveElementEventBinding);
+            doNotWatchEvents.push(eebFactory.findElementEventBindingData(EventID.MOUSE_ENTER));
+            doNotWatchEvents.push(eebFactory.findElementEventBindingData(EventID.MOUSE_LEAVE));
         }
         if (Options.getDoNotWatchKeyboardShortcutEvents()) {
-            doNotWatchEvents.push(KeystrokeElementEventBinding);
+            doNotWatchEvents.push(eebFactory.findElementEventBindingData(EventID.KEYSTROKE));
```

Comment:
Again a note here, also for the Test of this class, I'm changing this in another branch. :)